<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePass</class>
 <widget class="QDialog" name="ChangePass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Pass</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/splash/xilften.png</normaloff>:/splash/xilften.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QPushButton" name="Back">
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(204, 0, 0);
   	border-radius:5px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 255, 255);
}
</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/icons/icons/back.png</normaloff>:/icons/icons/back.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(185, 0, 0);</string>
         </property>
         <property name="text">
          <string>Change Pass</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="username">
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(255, 255, 255);
border-radius:5px;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="atualPass">
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(255, 255, 255);
border-radius:5px;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Actual Password</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newPass">
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(255, 255, 255);
border-radius:5px;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>New Password</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Confirm">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(204, 0, 0);
   	border-radius:5px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 255, 255);
}
</string>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/icons/icons/back.png</normaloff>:/icons/icons/back.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="changeOK">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(185, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
